### PR TITLE
Set up react and add components

### PR DESCRIPTION
**Pull request by alonsofl**
### Summary
Setup project and add components

#### Details
- Added React and removed installation boilerplate.
- Added linters.
- Added Redux (with folder structure).
- Added header component with a navigation bar.
- Added book list component.
- Added form component to add books.

> [Live Demo](https://bookstore-alonsofalconi.netlify.app/)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import './index.css';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
+  </React.StrictMode>,
+  document.getElementById('root'),
```
Comment (arslanbisharat) [inline src/index.js:13]: - [ ] Please make sure the index has 2 `<Link>`s in the navbar which leads to the booklist and categories locations.
The navbar might be in its own component, that's ok!


- [ ] Please make sure the index has 2 `<Route>`s, one for the BookList component and the other for the Categories component. In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page

 


I can see you have implemented this in App.j please move that here for now.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] All the unnecessary files should be removed to make your code clean.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (alonsofl) [reply src/index.js:13]: Hello @arslanbisharat, I have two `<NavLink>` in the Header component and they can be used interchangeably. It is necessary to change it to `<Link>`?

Comment (alonsofl) [reply src/index.js:25]: The issues have been fixed.
- The `<NavLink>` tags have been changed to `<Link>` in the Header.js component.
- `<Route>`'s have been moved to index.js.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by alonsofl** [comment]
Thanks, @arslanbisharat!

**Pull request merged**